.jenkins

```diff
@@ -21,12 +21,11 @@ pipeline {
                         }
                     }
                     environment {
-                        OPTIONS = '--with-options=compiler_warnings --arch=Volta70 --compiler=/usr/local/bin/clang++ --with-cuda --with-cuda-options=enable_lambda'
+                        OPTIONS = '-DCMAKE_BUILD_TYPE=Release -DKokkos_ENABLE_COMPILER_WARNINGS=ON -DKokkos_ARCH=Volta70 -DCMAKE_CXX_COMPILER=clang++ -DKokkos_ENABLE_CUDA=ON -DKokkos_ENABLE_CUDA_LAMBDA=ON -DKokkos_ENABLE_TESTS=ON'
                     }
                     steps {
-                        sh 'ln -s /usr/bin/ccache /usr/local/bin/clang++'
                         sh 'ccache --zero-stats'
-                        sh 'rm -rf build && mkdir -p build && cd build && ../generate_makefile.bash $OPTIONS && make test -j8'
+                        sh 'rm -rf build && mkdir -p build && cd build && cmake -DCMAKE_CXX_COMPILER_LAUNCHER=ccache $OPTIONS .. && make -j8 && ctest --output-on-failure'
```

Comment:
You are not reusing `OPTIONS` anywhere, are you? Can't you just put it here then? Then you would also not have to decide what to out in there which is currently slightly inconsistent between stages.
Alternatively, it would make sense to have all the `Kokkos` related options in `OPTIONS` and have the usual `CMake` variables separate.